tests/std/tests/P0896R4_ranges_alg_copy_if/test.cpp

```diff
@@ -7,18 +7,18 @@
 #include <compare>
 #include <concepts>
 #include <ranges>
-#include <utility>
 
 #include <range_algorithm_support.hpp>
 
+using namespace std;
 template <class T, class U>
 struct not_pair {
     T first;
     U second;
 
     auto operator<=>(const not_pair&) const = default;

```

Comment:
Pre-existing inconsistent `const`:
```suggestion
    auto operator<=>(not_pair const&) const = default;
```